src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
+
+  if (days < 3) {
+    return 40 * days;
+  }
+
+  if (days >= 3 && days <= 6) {
+    sum = 40 * days - 20;
+
+    return sum;
+  }
+
+  if (days >= 7) {
+    sum = 40 * days - 50;
```

Comment:
Again, replace the literals `40` and `50` with named constants to improve code readability and maintainability, in line with checklist item 2.